src/test/rustdoc-gui/headings.goml

```diff
@@ -15,7 +15,7 @@
 goto: file://|DOC_PATH|/test_docs/struct.HeavilyDocumentedStruct.html
 
 assert-css: ("h1.fqn", {"font-size": "24px"})
-assert-css: (".main-heading", {"border-bottom-width": "1px"})
+assert-css: (".main-heading", {"border-bottom-width": "0px"})
```

Comment:
I think you simply remove these lines. There is no point to check if something doesn't exist in this case.